src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -141,10 +142,10 @@
  *
  * import java.io.Serializable;
  *
- * class Foo {
- *   public static final int FOO_CONST = 1;
+ * class FirstClass {
+ *   public static final int FirstClassVariable = 1;
  *
- *   public void foo() {}
+ *   public void ClassMethod() {}
```

Comment:
Make all fields amd method to start from lower case.
It is well used rule in Java. Let's not be weird here.

Reply:
Respective fixes have been done @romani . Thanks for pointing out.

Reply:
Hello, are there any other changes to be made. Could the examples be better?@romani